Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM golang:alpine3.9 as builder
 
 RUN apk --update upgrade \
-    && apk --no-cache --no-progress add make git \
-    && rm -rf /var/cache/apk/*
+    && apk --no-cache --no-progress add make git
 
 WORKDIR /go/src/github.com/go-acme/lego
 COPY . .
 RUN make build
 
 FROM alpine:3.9
 RUN apk update \
```

Comment:
Understanding that this PR was to address the `tzdata`, I'm still worried a bit about the `--update` flag. The sideaffect to this is that it prevents reproducible builds, as remote libraries could be updated against the same alpine version affecting the build.

I'll approve for now, but leaving this here as a note for the future to possibly adjust.